aws-ssm-patchbaseline/build.gradle

```diff
@@ -0,0 +1,78 @@
+buildscript {
```

Comment:
is this internal as well?
please double check other resources as well, i don't remember exactly which files are internal, 
so i rely on you to do the due diligence that  all files here are public!!

Reply:
The build.gradle files is not an internal usage file. This file is used when we try to build the project. The other projects doesnot have this file, so i have removed the changes in repo